src/pybind/mgr/dashboard/tests/test_access_control.py

```diff
@@ -556,6 +556,17 @@ def test_set_user_password(self):
                                       'admin@user.com')
         self.assertGreaterEqual(user['lastUpdate'], user_orig['lastUpdate'])
 
+    def test_unicode_password(self):
+        self.test_create_user()
+        password = '您好'
```

Comment:
Haha, I was expecting something more... intriguing :P

Reply:
like
```py
password = u'章鱼不是密码'
```
? in English, it's "octopus is not the password".

also, shall we create a temporary file instead?

Reply:
@tchaikov Done.